packages/components/addon/components/hds/stepper/stepper-indicator/index.js

```diff
@@ -0,0 +1,103 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+// export const DEFAULT_STATUS = 'incomplete'
+// export const DEFAULT_TYPE = 'step'
+// export const DEFAULT_NUMBER = '1'
+
+export const STATUSES = ["incomplete", "inProgress", "complete"];
```

Comment:
Tip: if you run `yarn lint`, you'll see these issues before you push. 

There are some other options available as well (see package.json for the full list):

- `yarn lint:hbs` this will lint handlebars (`.hbs`) files
- `yarn lint:hbs --fix` this will lint `.hbs` files and also auto fix anything that is auto-fixable
- `yarn lint:js` this will lint `.js` files
- `yarn lint:js --fix` this will lint and also fix any issues that are auto-fixable

Don't worry if you forget, I often have a second commit that is "fixing linter" related 👍